tests/model/test_model.py

```diff
@@ -3,6 +3,7 @@
 from typing import Any
 
 import pytest
+import urwid
```

Comment:
This use of urwid in the Model makes me question the `ExitMainLoop` being handled here, as I said in the comment on the previous commit. Also, does the addition of this test belong with changes made to add `loading` variable? It doesn't seem obviously connected.